app/views/sign_up/registrations/new.html.erb

```diff
@@ -2,65 +2,63 @@
 
 <%= render 'shared/sp_alert' %>
 
-<%= render AlertComponent.new(
-      type: :error,
-      class: 'email-invalid-alert margin-bottom-4 display-none',
-      message: t('sign_up.email.invalid_email_alert_head'),
-    ) %>
-
 <h1 class='margin-y-0'><%= t('titles.registrations.new') %></h1>
 
-<div class='margin-bottom-6'>
-  <%= validated_form_for(
-        @register_user_email_form,
-        html: { autocomplete: 'off' },
-        url: sign_up_register_path,
-      ) do |f| %>
-    <%= f.input :email,
-                label: t('forms.registration.labels.email'),
-                required: true,
-                input_html: { autocorrect: 'off', aria: { invalid: false } } %>
-
-    <span class='email-invalid-alert-inline usa-error-message margin-top-neg-3 margin-bottom-3 display-none' role='alert'>
-      <%= t('sign_up.email.invalid_email_alert_inline') %>
-    </span>
+<%= simple_form_for(
+      @register_user_email_form,
+      html: { autocomplete: 'off' },
+      url: sign_up_register_path,
+    ) do |f| %>
+  <%= render ValidatedFieldComponent.new(
+        form: f,
+        name: :email,
+        as: :email,
+        label: t('forms.registration.labels.email'),
+        required: true,
+        input_html: { autocorrect: 'off' },
+        error_messages: {
+          typeMismatch: t('sign_up.email.invalid_email'),
+        },
```

Comment:
Maybe we could continue a [trend](https://github.com/18F/identity-idp/blob/fb3792f392b5290b24e7b4ec97788997860bd9c6/app/components/validated_field_component.rb#L22-L29) of inferring this error message based on the inferred type of the input?

Reply:
I think inferring it in the component implementation makes the most sense to me. I think unless we're expecting different email errors across the IDP that might be another way to simplify it.

Reply:
> I think inferring it in the component implementation makes the most sense to me. I think unless we're expecting different email errors across the IDP that might be another way to simplify it.

Yeah, in that case, it might also make sense to move the string from `sign_up.email.invalid_email` to something a bit more generic, similar to `forms.validation.required_checkbox` for the common required checkbox message.

Reply:
Updated in 4811ff069.